tasking/filters.py

```diff
@@ -1,10 +1,8 @@
-# -*- coding: utf-8 -*-
```

Comment:
Could you comment on what necessitated the changes in this module?

Reply:
Initially thought this was a python2 feature.

Reply:
The new `django-filter` doesn't support passing lists to `lookup_expr` like we used to do before. Needed to create a separate filter backend `TaskOccurrenceFilter` to handle the filtering of task occurrences from the tasks endpoint.